release/release.go

```diff
@@ -622,6 +679,40 @@ func LatestRC(ctx context.Context, owner, repo, k8sVersion string, client *githu
 
 }
 
+// rke2ChartVersion will return the version of the rke2 chart from the chart versions file
+func rke2ChartsVersion(branchVersion string) (map[string]chart, error) {
+
+	chartsMap := make(map[string]chart)
```

Comment:
Can you remove the empty line above?

```suggestion
	chartsData := make(map[string]chart)
```